parachain/README.md

```diff
@@ -65,6 +65,32 @@ For an example configuration, consult the [setup script](https://github.com/Snow
 
 To run the parachain tests locally, use `cargo test --release`. For the full suite of tests, use `cargo test --release --features runtime-benchmarks`.
 
+### Updating test data for inbound channel unit tests
```

Comment:
This feels like it belongs in the `ethereum` sub folder's README?

Reply:
The ethereum README has a section about updating test data for its unit tests (using E2E tests), and this is the equivalent for updating the parachain unit test data (using ethereum tests). It probably feels like it should move because it's using tests from `ethereum` to generate fixture data, but its purpose is to help update the parachain unit test data. Using the ethereum tests is a detail: if we started getting the data elsewhere (eg. E2E tests), we'd keep this in the parachain README.